.github/workflows/nightly.yml

```
@@ -38,6 +38,7 @@ jobs:
 
   update-vision-commit-hash:
     uses: ./.github/workflows/_update-commit-hash.yml
+    if: ${{ github.event_name != 'push' }}
```

Wouldn't it be better to say where it should go, rather then where it shouldn't?
```suggestion
    if: ${{ github.event_name == 'schedule' }}
```